07_errors/alextmz/errors.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import "fmt"
+
+type Error struct {
+	Message string
+	Code    int
+}
+
+const (
+	Err400BadRequest     = 400
+	Err404NotFound       = 404
+	Err409Conflict       = 409
+	Err500InternalError  = 500
+	Err501NotImplemented = 501
```

Comment:
use meaningful errors / error classification and map to http status code if/when/where needed. http status codes should not be your dimension of classification.